doc/OpenMPSupport.md

```diff
@@ -0,0 +1,220 @@
+# TAFFO OpenMP Support
+TAFFO has preliminary support for [OpenMP](https://www.openmp.org) parallel constructs.
```

Comment:
```suggestion
TAFFO has preliminary support for [OpenMP](https://www.openmp.org) parallel constructs.
We classify the support for most popular OpenMP features in **fully supported**, **partially supported**, and **not supported**.
```

Reply:
After the latest _improvements_ (automatic disabling of conversion for shared variables), the **partially supported** (which had enormous problems) and the **not_supported** sets are now merged into one, therefore I would avoid doing further divisions. 

The main difference, which I will better explain in the user guide, is:
- supported parallel constructs have full support for taffo conversions of variables
- unsupported parallel constructs do not break anything, but no shared variable is converted in fixed-points